packages/platform/example/lib/main.dart

```diff
@@ -0,0 +1,149 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:platform/platform.dart';
+
+/// This sample app shows how to single view the platform details.
```

Comment:
I'm having trouble parsing this; what does "singe view" mean here?

Reply:
this has been changed